### DISCUSSION TITLE
Welcome to SCIRun Discussions!

### DISCUSSION THREAD

**Discussion by dcwhite** [SCIInstitute/SCIRun#2179] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dcwhite** [comment]
Hi, I'm the main developer of SCIRun.

**Comment by jessdtate** [comment]
Do we need rules of conduct?

**Reply by dcwhite** [reply]
We could just copy Github's https://github.com/gh-community/discussions-feedback/blob/master/CODE_OF_CONDUCT.md

**Reply by dcwhite** [reply]
https://www.contributor-covenant.org/

**Reply by dcwhite** [reply]
Ooh threaded replies!

**Reply by tpat95** [reply]
I vote for anarchy. Let's give all users admin rights!
